src/Elastic.Apm/Model/Payload/Transaction.cs

```diff
@@ -95,6 +95,9 @@ public ISpan StartSpan(string name, string type, string subType = null, string a
 
 		internal Span StartSpanInternal(string name, string type, string subType = null, string action = null)
 		{
+			if (name.Length > 1024)
```

Comment:
Could we move this into the setter of 'Span.Name'? I'd also move `1024` into the `Consts` class. Who knows...maybe we change this :)